### PR TITLE
Remove support for non-strong mode & add "grind deploy" to match front-end

### PR DESCRIPTION
**Pull request by jcollins-g**
Just a cleanup; should have no impact on Dartpad's behavior.

### CODE REVIEW

**Comment by jcollins-g** [comment]
Travis doesn't like this, investigating.

File: tool/fuzz_driver.dart [hunk 1]
```diff
@@ -247,10 +239,10 @@ Future<num> testAnalysis(
   lastOffset = null;
   if (_SERVER_BASED_CALL) {
     await withTimeOut(server.analyzeGet(source: src));
-    await withTimeOut(server.analyzeGet(source: src, strongMode: true));
+    await withTimeOut(server.analyzeGet(source: src));
```
Comment (devoncarew) [inline tool/fuzz_driver.dart]: I think we can delete this line (it duplicates the one above now).

File: tool/fuzz_driver.dart [hunk 2]
```diff
@@ -247,10 +239,10 @@ Future<num> testAnalysis(
...
   } else {
     await withTimeOut(analysisServer.analyze(src));
-    await withTimeOut(strongAnalysisServer.analyze(src));
+    await withTimeOut(analysisServer.analyze(src));
```
Comment (devoncarew) [inline tool/fuzz_driver.dart]: ditto

File: tool/warmup.dart [hunk 3]
```diff
@@ -11,7 +11,7 @@ String uri;
 
 const BASE_URI = "dart-services.appspot.com/api/dartservices/v1/";
 // const BASE_URI = "localhost:8080/api/dartservices/v1/";
-const count = 100;
+const count = 200;
```
Comment (devoncarew) [inline tool/warmup.dart]: I'm curious why the increase?

File: doc/generated/_dartpadsupportservices.dart [hunk 4]
```diff
@@ -23,31 +25,29 @@ class P_dartpadsupportservicesApi {
...
-   *
-   * Completes with a [UuidContainer].
-   *
-   * Completes with a [commons.ApiRequestError] if the API endpoint returned an
-   * error.
-   *
-   * If the used [http.Client] completes with an error when making a REST call,
-   * this method will complete with the same error.
-   */
+  /// Store a gist dataset to be retrieved.
```
[the remark this answers is not in the document]
Comment (jcollins-g) [reply doc/generated/_dartpadsupportservices.dart]: Yes.  grind deploy will now automatically regenerate these.  Functionally they should be equivalent to the old version, for now.

File: tool/fuzz_driver.dart (hunk 1 above)
Comment (jcollins-g) [reply tool/fuzz_driver.dart]: done

File: tool/fuzz_driver.dart (hunk 2 above)
Comment (jcollins-g) [reply tool/fuzz_driver.dart]: done

File: tool/warmup.dart (hunk 3 above)
Comment (jcollins-g) [reply tool/warmup.dart]: It's typically required more than one run of the warmup script to warm up the backend, so just doubling it.

**Pull request merged**